storage/bucket_test.go

```diff
@@ -60,6 +60,7 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 		Encryption: &BucketEncryption{DefaultKMSKeyName: "key"},
```

Comment:
Let's make sure we have test coverage for the proto converters as well. Also, you should make an integration test to cover the roundtrip with the service (both create and update).

Reply:
Yep was meaning to write an integration test... also added 2 tests for the proto converters. @tritone PTAL, thanks!

Reply:
Thanks for adding these additional tests; didn't realize we were missing these unit tests for proto converter methods. We should consider auditing and filling others in if they are missing. @noahdietz @BrennaEpp